install/cluster-operator/033-ClusterRole-strimzi-zookeeper-cluster.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: strimzi-zookeeper-cluster
```

Comment:
We do not need this role and Zookeeper doesn't need these rights. The same applies to mirror maker and connect. You should remove these new RBAC files and just create the Service Accounts. This is just about isolating the applications by giving them service account without any actual rights.

Reply:
Ah, thanks. Should be fixed now.